r/R/compression.R

```diff
@@ -37,10 +37,14 @@ compression_codec <- function(type = "GZIP") {
 
 #' Compressed output stream
 #'
+#' @details This function is not supported in Windows.
+#'
 #' @param stream Underlying raw output stream
 #' @param codec a codec
 #' @export
 CompressedOutputStream <- function(stream, codec = compression_codec("GZIP")){
+  if (.Platform$OS.type == "windows") stop("'CompressedOutputStream' is unsupported in Windows.")
```

Comment:
Is this because of missing compression libraries?

Reply:
That is correct. We can probably solve this by figuring out which are the missing dependencies and adding support to compile them in Mingw, but I much rather suggest we disable this functionality in the 0.13 in Windows.